apm-server.yml

```diff
@@ -240,6 +240,10 @@ apm-server:
       # Sourcemapping is enabled by default.
       #enabled: true
 
+      # Maximum allowed line length in characters. Default maximum line length
+      # is unlimited, indicated by `0`. Minimum allowed value is 16.
+      #max_line_length: 0
```

Comment:
Are these options listed elsewhere and the apm-server.yml files generated? or do I add this to e.g. apm-server.docker.yml as well?

Reply:
Both `apm-server.yml` and `apm-server.docker.yml` are generated from `_meta/beat.yml`. Edit that one and run `make update` (or just `make config`).